### PR TITLE
MAVExplorer - spaces now allowed in log file name and path

### PR DESCRIPTION
**Pull request by stephendade**

### CODE REVIEW

File: MAVProxy/tools/MAVExplorer.py [hunk 1]
```diff
@@ -428,9 +429,13 @@ def cmd_param(args):
 def cmd_loadfile(args):
     '''callback from menu to load a log file'''
     if len(args) != 1:
-        print "Error loading file"
+        fileargs = " ".join(args)
```
Comment (peterbarker) [inline MAVProxy/tools/MAVExplorer.py]: Should we allow quoting of the filename here instead?  Use shlex to break the line up for us?  Taking logically separate arguments and squishing them together is a little awkward.

File: MAVProxy/tools/MAVExplorer.py [hunk 2]
```diff
@@ -446,6 +451,14 @@ def loadfile(args):
     load_graphs()
     setup_menus()
 
+def shlex_quotes(value):
```
Comment (peterbarker) [inline MAVProxy/tools/MAVExplorer.py]: Perhaps `shlex_split` instead?

Comment (stephendade) [inline MAVProxy/tools/MAVExplorer.py]: Yep, will fix that

File: MAVProxy/tools/MAVExplorer.py (hunk 1 above)
Comment (stephendade) [inline MAVProxy/tools/MAVExplorer.py]: cmd_loadfile should only expect 1 argument anyway - the name of the logfile to be loaded. Not sure why multiple arguments should be expected here.

**Comment by tridge** [comment]
applied with rebase fixup, thanks!

**Pull request closed without merging**